envoy-curses/src/clusters_pane.js

```diff
@@ -0,0 +1,261 @@
+/* eslint camelcase: ["error", {properties: "never"}]*/
+
+const blessed = require('blessed')
+const Theme = require('./theme')
+const Box = blessed.Box
+const contrib = require('blessed-contrib')
+const Menu = require('./menu.js')
+const theme = require('./theme.js')
+
+class ClustersPane extends Box {
+  constructor(options) {
+    super(options)
+    this.options = options || {}
+    this.screen = options.screen
+    this.style = this.options.style || Theme.style.base
+    this.clusters = options.clusters
+    this.stats = options.stats
+    this.log = options.log
+    this.chartedStat = 'rq_total'
+    this.availableStats = []
+    this.selectedClusterName = ''
+    this.tableData = {
+      headers: ['cluster', 'cx act', 'rq act', 'rq total', 'members', 'healthy'],
```

Comment:
it would be cool if the currently selected stat was added to the table for all clusters. maybe a TODO?

Reply:
Made this change